### DISCUSSION TITLE
After increase disc and update nextcloud containers in docker the nextcloud database container doesn't start

### DISCUSSION THREAD

**Discussion by Petrik-a-dost** [nextcloud/all-in-one#2146] [Questions] [answered]
Hi All,

I have Nextcloud AIO v4.5.0 in docker on Ubuntu server.
I have big issue with container nextcloud database. After increase disc and update all nextcloud containers images of nextcloud AIO in docker the container nextcloud-database doesn’t start.
Log of nextcloud-database container is:

```
Database export failed the last time. Most likely was the export time not high enough.
Please report this to [Issues · nextcloud/all-in-one · GitHub](https://github.com/nextcloud/all-in-one/issues). Thanks!
Database export failed the last time. Most likely was the export time not high enough.
Please report this to [Issues · nextcloud/all-in-one · GitHub](https://github.com/nextcloud/all-in-one/issues). Thanks!
Database export failed the last time. Most likely was the export time not high enough.
Please report this to [Issues · nextcloud/all-in-one · GitHub](https://github.com/nextcloud/all-in-one/issues). Thanks!
```

I am stuck on this, because nextcloud doesn’t start because nextcloud-database doesn’t start.
Could you somebody help me please with this issue?

THX, Petr

I am stuck on this screen:
<img width="521" alt="image" src="https://user-images.githubusercontent.com/5670273/224513026-bdba296e-8c7c-409a-aa96-5e586c6d99f9.png">

**Comment by szaimen** [comment]
Hi, please restore a backup. This should fix the problem.

**Reply by Petrik-a-dost** [reply]
> Hi, please restore a backup. This should fix the problem.

I tried it and it didn't help :(. I haven't any important data there, because I had new instalation of Nextcloud. I tried to delete all nextcloud containers and images via Portainer and tries to install again, but it didn't work and I think that i cannot reinstall nextcloud-mastercontainer because after deleting all containers of nextcloud and install again nextcloud-mastercontainer remember previous setting and show me screen with containers like in my post. I think that best solution for me is delete all containers and install again but mastercontainer I cannot reinstall. Maybe that on my disc is located some persist file, but i don't know where.

**Comment by szaimen** [comment, marked as answer]
Here is how to reset AIO properly:

https://github.com/nextcloud/all-in-one#how-to-properly-reset-the-instance
